### ISSUE TITLE
subdirectories of src

### ISSUE THREAD

**Issue by kbuzzard** [leanprover-community/format_lean#7]
I have a Lean project, with files `sandwich.lean` and `src/M40001/sandwich.lean` and these are identical files. I'm in the root directory of the project (the directory which contains `sandwich.lean`). 

1) `format_lean`. If I type
```
format_lean --inpath sandwich.lean --outdir html --lib-path _target/deps/mathlib/src/
```

then everything works fine. The contents of the `html` directory are
```
colorful.css   lecture.css      teach_lean.css      with_tufte.css.map
jquery.min.js  lecture.css.map  teach_lean.css.map
lean.js        sandwich.html    with_tufte.css
```

But if I type
```
format_lean --inpath src/M40001/sandwich.lean --outdir html --lib-path _target/deps/mathlib/src/
```

I get an error
```
FileNotFoundError: [Errno 2] No such file or directory: 'html/src/M40001/sandwich.html'
```
[note this is about html files] and if I try 
```
format_lean --inpath M40001/sandwich.lean --outdir html --lib-path _target/deps/mathlib/src/
```

I get `No such file or directory: 'M40001/sandwich.lean'` [note this is about lean files] (and indeed there is no such file or directory as I am in the root directory).

2) Now I move the root `sandwich.lean` to `src` and I add `format.toml` in root containing the line `only = ['sandwich.lean']`. I type `format_project` and things work fine, although I will remark that my html directory looks like this now:
```
colorful.css   lean.js      lecture.css.map
jquery.min.js  lecture.css  sandwich.html
```

(far fewer files). It works fine.

But I still have this identical copy of sandwich.lean in `src/M40001`. If I edit `format.toml` so it says `only = ['M40001/sandwich.lean']` and run `format_project`, I get an html file but nothing is clickable (i.e. I can't "open" the web page to see the tactics).

**Comment by PatrickMassot** [comment]
I should remove the `format_lean` command, it's mostly for testing purposes. The real thing is `format_project`. There I can't reproduce your problem. Could you post a full archive of a minimal example?

**Comment by kbuzzard** [comment]
Jason's repo https://github.com/JasonKYi/M4000x_LEAN_formalisation used to work but now format_project is giving us html files which aren't clickable. Maybe there's some syntax error somewhere but we have no idea how to debug.

**Comment by PatrickMassot** [comment]
I'm sorry I can't reproduce that problem. I just cloned the repository, `leanpkg configure`, `update-mathlib` and `format_project` and everything works except that the broken link tag at https://github.com/JasonKYi/M4000x_LEAN_formalisation/blob/master/src/M40001/M40001_C2.lean#L27 (a `<a>` should be `</a>`) messes up the rendering in Firefox (everything gets underlined after that point). By the way, you can use markdown links.

**Comment by kbuzzard** [comment]
I clone the repo, leanpkg configure, update-mathlib,  format_project, change directory into html, start a web server in this directory, and then navigate to M40001 and I have a bunch of unclickable stuff in the html files. How can I debug this? I'm on Ubuntu 18.04.

**Comment by PatrickMassot** [comment]
Fixed in 1122e2ba9d04942b2193d1167c16e89c7dfe2e02
